libraries/rush-sdk/.npmignore

```diff
@@ -27,4 +27,7 @@
 # DO NOT MODIFY THE TEMPLATE ABOVE THIS LINE
 #--------------------------------------------
 
+/lib-shim/**
+!/lib-shim/index.js
```

Comment:
`!/lib-shim/index.js` is a bit arbitrary. I could see someone expanding this logic and introducing a helper file, and then publishing a broken release because they didn't imagine that `lib-shim/helper.js` would get ignored differently from `lib-shim/index.js`.

Reply:
Should we switch to using the `files` field in package.json (which is a much clearer allow list of exactly what should be included)?